pcdsdevices/lasers/ek9000.py

```diff
@@ -19,12 +19,15 @@ class El3174AiCh(Device):
         The PV base of the card.
     """
     measured = Cpt(EpicsSignalRO, '.VAL', kind='normal')
-    _raw_adc = Cpt(EpicsSignalRO, '.RVAL', kind='config')
+    raw_adc = Cpt(EpicsSignalRO, '.RVAL', kind='config')
     egu = Cpt(EpicsSignal, '.EGU', kind='config')
-    _egu_max = Cpt(EpicsSignal, '.EGUF', kind='config')
-    _egu_min = Cpt(EpicsSignal, '.EGUL', kind='config')
-    _conversion = Cpt(EpicsSignal, '.LINR', kind='config')
-    _precision = Cpt(EpicsSignal, '.PREC', kind='config')
+    # TJ: These may be usefult late, but not now
+    # egu_max = Cpt(EpicsSignal, '.EGUF', kind='config')
+    # egu_min = Cpt(EpicsSignal, '.EGUL', kind='config')
+    eslo = Cpt(EpicsSignal, '.ESLO', kind='config')
```

Comment:
Could consider some more user-friendly names like `slope` and `offset`, perhaps.

Also would suggest including a `doc=''` kwarg for such components if you have useful contextual information to be documented.

Reply:
Yeah, good idea. I also note a few egregious typos now that I look again... 🤦 

I need to get in the habit of using doc kwargs anyway.